### PR TITLE
+ Adding a new Server

### PR DESCRIPTION
**Pull request by Kallecrafter**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/brickcraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "direct_ip": "brickcraft.eu",
+    "supported_languages": [
+        "de"
+    ],
+    "social": {
+        "discord": "https://discord.gg/ThCt8Pq",
+        "teamspeak": "craftergang.de",
+        "tiktok": "craftergang",
+        "youtube": "https://www.youtube.com/@craftergang",
+        "instagram": "craftergang"
```
Comment (maaaathis) [inline minecraft_servers/brickcraft/manifest.json:13]: Are you sure that these are your social profiles?

File: minecraft_servers/brickcraft/icon@2.png
Comment (maaaathis) [inline minecraft_servers/brickcraft/icon@2.png:1]: file has to be named `icon@2x.png`

**Review by maaaathis — CHANGES_REQUESTED** [review]

**Review by Kallecrafter — COMMENTED** [review]
fixed

File: minecraft_servers/brickcraft/manifest.json (hunk 1 above)
Comment (Kallecrafter) [reply minecraft_servers/brickcraft/manifest.json]: fixed

**Review by Kallecrafter — COMMENTED** [review]
fixed

File: minecraft_servers/brickcraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "nice_name": "BrickCraft",
+    "direct_ip": "brickcraft.eu",
+    "supported_languages": [
+        "de"
+    ],
+    "gamemodes": {
+        "skypvp": {
+            "name": "SkyPvP",
+            "color": "#0d6914",
+            "command": "/spawn",
```
Comment (DoJapHD) [inline minecraft_servers/brickcraft/manifest.json:12]: That's not really a command to "join" the gamemode?

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/brickcraft/icon@2.png
Comment (Kallecrafter) [reply minecraft_servers/brickcraft/icon@2.png:1]: yes

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**